### PR TITLE
Add nxos_lacp_interfaces model

### PR DESCRIPTION
**Pull request by trishnaguha**
Signed-off-by: Trishna Guha <trishnaguha17@gmail.com>

### CODE REVIEW

File: models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  module: nxos_lacp_interfaces
+  version_added: 2.9
+  short_description: Manage Link Aggregation Control Protocol (LACP) attributes of interfaces on Cisco NX-OS devices.
+  description: This module manages Link Aggregation Control Protocol (LACP) attributes of NX-OS Interfaces.
+  author: Trishna Guha (@trishnaguha)
+  notes:
+    - Tested against <network_os> 7.3.(0)D1(1) on VIRL
+  options:
+    config:
+      description: LACP configurations for interface.
```
Comment (samccann) [inline models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml]:
```suggestion
      description: A dictionary of LACP interfaces options.
```

File: models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        suspend_individual:
+          description:
+            - port-channel lacp graceful convergence.Disable this only with lacp ports
+              connected to Non-Nexus peer. Disabling this with Nexus peer can lead
+              to port suspension. Applicable only for Port-channel.
+          type: bool
+        convergence:
+          description:
+            - This dict contains configurable options related to max and min port-channel links.
+              Applicable only for Port-channel.
```
Comment (samccann) [inline models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml]: is this description correct? seems a repeat of the links description

File: models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+          type: dict
+          suboptions:
+            graceful:
+              description:
+                - port-channel lacp graceful convergence.Disable this only with lacp ports 
+                  connected to Non-Nexus peer. Disabling this with Nexus peer can lead
+                  to port suspension.
```
Comment (samccann) [inline models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml]: similarly - same description as suspend_individual...

File: models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml (hunk 2 above)
Comment (trishnaguha) [reply models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml]: updated

File: models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml (hunk 3 above)
Comment (trishnaguha) [reply models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml]: updated

**Pull request merged**